src/js/components/productSupplier/create/InvalidItemsIndicator.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiCheckboxCircleLine } from 'react-icons/all';
+
+import Translate from 'utils/Translate';
+
+const InvalidItemsIndicator = ({ className, errorsCounter }) => (
+  <div className={`invalid-items-indicator ${className}`}>
+    <span>
```

Comment:
This should be a button that will filter rows in the table

Reply:
this button/indicator is handled in another ticket, I took it as the next one that I am working on